pkg/cmd/run.go

```diff
@@ -16,19 +16,25 @@ import (
 	"github.com/cenkalti/backoff/v4"
 	"github.com/openfga/openfga/internal/build"
 	"github.com/openfga/openfga/pkg/cmd/service"
+	cmdutil "github.com/openfga/openfga/pkg/cmd/util"
 	"github.com/openfga/openfga/pkg/logger"
 	"github.com/spf13/cobra"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
 )
 
 func NewRunCommand() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "run",
 		Short: "Run the OpenFGA server",
 		Long:  "Run the OpenFGA server.",
 		Run:   run,
+		Args:  cobra.NoArgs,
```

Comment:
Right now you can run `openfga run blah` and it accepts it. This makes is to that you get an error:

```
➜  ./openfga run blah
Error: unknown command "blah" for "openfga run"
Usage:
  openfga run [flags]
```